lesson6.js

```diff
@@ -1,7 +1,13 @@
 const ul = document.getElementById('js-list');
+const loadingTime = document.createElement('img');
+loadingTime.src = "loading-circle.gif";
+loadingTime.id = 'loading-img'
+ul.appendChild(loadingTime);
+const loading = document.getElementById('loading-img');
+
 const info = [
-  {to: "bookmark.html", img: "1.png", alt:"画像1", text: "ブックマーク"},
-  {to: "message.html", img: "2.png", alt:"画像2", text: "メッセージ"}
+  {to: "bookmark.html", img: "bookmark-icon.png", alt:"画像1", text: "ブックマーク"},
+  {to: "message.html", img: "mail-icon.png", alt:"画像2", text: "メッセージ"}
   ];
 const fragment = document.createDocumentFragment();
```

Comment:
Oh sorry, it's not clearly what I said. 
I meant this time you can define it in a function.
As @443n0511 said [here](https://github.com/rikokko/to_be_frontend/pull/13#discussion_r721303045), it will reduce bugs.

I think you can define it inside of `makeLinkList().then((item)` :)

Reply:
I'm sorry I can't interpret it. I misunderstood this time and the next story. Thank you for your polite guidance.

Reply:
This one looks good to me!